### ISSUE TITLE
Show Description Character Limit on Project Submission Page

### ISSUE THREAD

**Issue by DBNess** [CodeMontageHQ/codemontage#273]
**Description**
On the project submission page at https://www.codemontage.com/organizations/new, there's a 255-character limit for the Description that is not reflected to the user at all.

![codemontage](https://cloud.githubusercontent.com/assets/226228/5254987/7a513fac-7984-11e4-8f61-16d3a1c3838c.jpg)

**Ask**
Enforce this length limitation on the frontend so that users get feedback if their descriptions are too long rather than triggering an unseeable database error.

**To Do**
There are a number of ways to go about doing this. Pick at least one of the following, and we should be good to go.
- [ ] Set a hard html limit on the input.
- [ ] Add a length validation on the `Project` model.
- [ ] Use the [ClientSideValidations gem](https://github.com/DavyJonesLocker/client_side_validations-simple_form#usage) to set a validation in the frontend.
- [ ] Add a dynamically-updated counter (e.g. 175/255).

**Comment by thefenry** [comment]
Yeah I think I can manage these tasks. I will try to get this in the next day or so.

**Comment by thefenry** [comment]
I am sorry I have been busy at work and moving. I tried getting this done the other day and I am having a hard time modifying the simple-fields. I cannot figure out how to add a html element after the label and before the input. I read some of the documentation but none of the solutions where working for me. I managed to do it through jquery but it is not very efficient since it does not load this until it finishes loading the page. Any suggestions would be appreciated. Or If anyone would like to take over it let me know.

**Comment by courte** [comment]
@thefenry - Congrats on the move! I hope you're settled in now.

I appreciate you taking this issue on. If the jQuery option is working - however inefficient - it would be great if you could submit it as a PR. If it doesn't work, then maybe just changing the placeholder would be be a clear marker. Any small step forward is really helpful to us.

Thanks again for taking the time! :smile: :tada:

**Comment by laramelniker** [comment]
Hi @thefenry , I know it's been a few months so I'd like to take over if you don't mind. 🐫🐩

**Comment by thefenry** [comment]
@laramelniker Yeah definetly go for it. I have been busy with other projects and forgot about this one.
